Makefile

```diff
@@ -128,15 +128,12 @@ endif
 
 XGO := $(strip $(if $(XGOOS),GOOS=$(XGOOS)) $(if $(XGOARCH),GOARCH=$(XGOARCH)) $(if $(XHOST_TRIPLE),CC=$(CC_PATH) CXX=$(CXX_PATH)) $(GO))
 
-.DEFAULT_GOAL := all
-.PHONY: all
-all: build test lint
```

Comment:
We include build/common.mk above, so the first rule has already been defined by the time we get here. We could put an additional target above the include $(REPO_ROOT)/build/common.mk line, but I'm not sure that buys us anything. If you're here, you've already waded through ~100 lines of platform-specific drudgery, so you're probably not afraid of Make.